### PR TITLE
Add swagger spec json file

### PR DESCRIPTION
**Pull request by JoshFen95**
Update readme to link to swagger file

See JIRA: https://companieshouse.atlassian.net/browse/BI-10680

### CODE REVIEW

**Review by ch-tstrickland — COMMENTED** [review]
Don't call them swagger specs anymore. They are OpenAPI specs.

File: spec/swagger.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  "servers": [
+    {
+      "url": "http://api.chs.local/siccode-api",
+      "description": "Development server"
+    }
+  ],
+  "paths": {
+    "/search": {
+      "post": {
+        "summary": "Search for sic code via economic activity",
```
Comment (ch-tstrickland) [inline spec/swagger.json:16]: sic -> SIC throughout the file (except in var names, obviously!)

File: spec/swagger.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        "operationId": "searchSicCodes",
+        "description": "Return all matching sic code records that correspond with the search terms. This will be ordered by most number of hits (for example, results with 2 keyword matches will be retuned before matches with 1 keyword match)",
```
Comment (ch-tstrickland) [inline spec/swagger.json:18]: Generally speaking, if numbers are less than 10, write them out. Thus 2 keyword -> two keyword and 1 keyword -> one keyword.
Typo:
retuned -> returned.

File: spec/swagger.json [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            "description": "If true then only return result where all keywords are found",
+            "schema": {
+              "type": "boolean"
+            },
+            "required": true
+          },
+          {
+            "in": "path",
+            "name": "context_id",
+            "description": "Unique ID to allow end-to-end logging in systems such as cloud watch (Recomended is `applicationId-uniqueNumber`)",
```
Comment (ch-tstrickland) [inline spec/swagger.json:41]: cloud watch -> CloudWatch

File: spec/swagger.json [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    "/search": {
+      "post": {
+        "summary": "Search for sic code via economic activity",
+        "operationId": "searchSicCodes",
+        "description": "Return all matching sic code records that correspond with the search terms. This will be ordered by most number of hits (for example, results with 2 keyword matches will be retuned before matches with 1 keyword match)",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "keywords",
+            "description": "Keywords for the economic activities to be used in the search",
```
Comment (ch-tstrickland) [inline spec/swagger.json:23]: One variable containing many values?
This seems to be one parameter whose key could point to many values in one string. If so, how are they separated?
Perhaps consider this instead:
https://stackoverflow.com/questions/62527254/how-does-open-api-3-0-support-a-single-query-param-key-with-multiple-values

Comment (ch-tstrickland) [reply spec/swagger.json:23]: Also, this is perhaps the one parameter that could do with an example.

Comment (JoshFen95) [reply spec/swagger.json:23]: Updated this variable to better illustrate it as a search string. See commit

File: spec/api-spec.json [hunk 5]
```diff
@@ -0,0 +1,111 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "Sic-Code-API",
+    "version": "1.0"
+  },
+  "servers": [
+    {
+      "url": "http://api.chs.local/siccode-api",
```
Comment (ch-tstrickland) [inline spec/api-spec.json:9]: I think that in the meeting with Steve, we talked about this becoming /sic-code-api

File: spec/api-spec.json [hunk 6]
```diff
@@ -0,0 +1,111 @@
+{
+  "openapi": "3.0.2",
+  "info": {
+    "title": "Sic-Code-API",
```
Comment (ch-tstrickland) [inline spec/api-spec.json:4]: If this is to follow the naming of the service example, perhaps:
sic-code-api
(the service names here always seem to be lowercase).

File: spec/api-spec.json (hunk 5 above)
Comment (JoshFen95) [reply spec/api-spec.json]: updated to /sic-code-search as per other services

File: spec/api-spec.json (hunk 6 above)
Comment (JoshFen95) [reply spec/api-spec.json]: updated to lowercase

**Review by ch-tstrickland — APPROVED** [review]
A superb spec.

**Pull request merged**